01-shapefile-attributes.Rmd

```diff
@@ -75,182 +79,292 @@ CSV to Shapefile in R</a>
 4. <a href="{{ site.baseurl }}/R/crop-extract-raster-data-R/">
 Crop and extract raster values in R</a>
 
-</div>
-
-NOTE: The data used in this tutorial were collected at Harvard Forest which is
-a the National Ecological Observatory Network field site <a 
-href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
-More about the NEON Harvard Forest field site</a>. These data are proxy data for
-what will be available for 30 years from the NEON flux tower [from the NEON data portal](http://data.neoninc.org/ "NEON data").
-{: .notice}
- 
-##Notes about R Libraries
+###Additional Resources
 
-To work with vector data in R, we can use the `rgdal` library. We will load the 
-`raster` library to work with rasters. The `raster` library also allows us to 
-explore metadata using similar commands with both rasters and vectors.
-
-##Setting up the workspace
-If you completed Lesson 00  {{site.baseurl}}R/open-shapefiles-in-R/  you can skip this code chunk.
-```{r load-libraries-data }
+</div>
 
-library(rgdal)
-library (raster)   #notice sp package should always load with raster. 
-#in case the libraries and data are not still loaded
+#Shapefile Metadata & Attributes
+When we imported a shapefile layer into `R`, the `readOGR()` function 
+automatically stores informatation about and associated with the data. The
```

Comment:
When we import a shapefile into R, the `readOGR()` function automatically stores metadata and attributes associated with the file.

please delete the rest - especially the non-geographic information part.